package.json

```diff
@@ -47,10 +47,12 @@
     "eslint-config-webpack": "^1.2.1",
     "eslint-plugin-import": "^2.2.0",
     "extract-text-webpack-plugin": "^3.0.0",
+    "fsevents": "^1.2.4",
     "jest": "^19.0.2",
     "lint-staged": "^3.4.0",
-    "lodash": "^4.17.4",
-    "node-sass": "^4.5.3",
+    "lodash": "^4.17.11",
+    "nan": "^2.11.1",
```

Comment:
This wasn’t a dependency before, but now it is. Seems unintentional?

Reply:
Previously an older version of `nan` was present in package-lock.json - one that led to compile failures - and I wasn't sure how to get it to update besides declaring the version directly here.

Reply:
npm update should do that. If it doesn’t add it here, run npm install, then remove it here again, and run npm install again